packages/terra-doc-template/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "terra-doc-template",
+  "main": "lib/DocTemplate.js",
+  "version": "0.0.0",
+  "description": "Provides a template for documentation pages",
```

Comment:
Lets update to make consistent: 
> The terra-doc-template component provides an easily adjustable template for documentation pages.

Reply:
I'd leave out the easily part IMO. I do think adding an adjective such as adjustable/customizable to template helps with the description though.